apps/condo/domains/billing/schema/ValidateQRCodeService.test.js

```diff
@@ -194,6 +207,29 @@ describe('ValidateQRCodeService', () => {
                 }])
             })
         })
+
+        test('should throw an error if no recipient in acquiring context', async () => {
+            const [o10n] = await createTestOrganization(adminClient)
+            const [property] = await createTestProperty(adminClient, o10n)
+            await addBillingIntegrationAndContext(adminClient, o10n, {}, { status: CONTEXT_FINISHED_STATUS })
+            await addAcquiringIntegrationAndContext(adminClient, o10n, {}, { status: CONTEXT_FINISHED_STATUS })
+            const qrCodeObj = generateQrCodeObj({
+                PayeeINN: o10n.tin,
+                PayerAddress: `${property.address}, кв. 1`,
+            })
+
+            const qrCodeString = stringifyQrCode(qrCodeObj)
+            await expectToThrowGQLError(
+                async () => validateQRCodeByTestClient(adminClient, { qrCode: qrCodeString }),
+                {
+                    mutation: 'validateQRCode',
+                    code: 'BAD_USER_INPUT',
+                    type: 'WRONG_FORMAT',
+                    message: 'Provided bank account is not in the system',
```

Comment:
maybe change error message to smth like No recipient for Acquiring Context?

Reply:
I'd like this error covers all cases (no context.recipient, no billing recipient, ...)